parity-crypto/Cargo.toml

```diff
@@ -24,6 +27,9 @@ aes = "0.3.2"
 aes-ctr = "0.3.0"
 block-modes = "0.3.3"
 pbkdf2 = "0.3.0"
+quick-error = "1.2.2"
```

Comment:
We've removed quick-error in #149, let's not reintroduce it again as it's deprecated.